wso2is_analytics/hieradata/dev/wso2/wso2is_analytics/pattern-1/default.yaml

```diff
@@ -0,0 +1,417 @@
+#    username: "%{hiera('wso2::datasources::mysql::username')}"
+#    password: "%{hiera('wso2::datasources::mysql::password')}"
+#    max_active: "%{hiera('wso2::datasources::common::max_active')}"
+#    max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
+#    test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
+#    default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
+#    validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
+#    validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
+#    default_auto_commit: false
+#    initial_size: 0
+#    test_while_idle: true
+#    min_evictable_idle_time_millis: 4000
+
+wso2::metrics_datasources:
+  wso2_metrics_db:
+    name: WSO2_METRICS_DB
+    description: The default datasource used for WSO2 Carbon Metrics
+    driver_class_name: "%{hiera('wso2::datasources::h2::driver_class_name')}"
+    url: jdbc:h2:repository/database/WSO2METRICS_DB;DB_CLOSE_ON_EXIT=FALSE;AUTO_SERVER=TRUE
+    username: "%{hiera('wso2::datasources::common::username')}"
+    password: "%{hiera('wso2::datasources::common::password')}"
+    jndi_config: jdbc/WSO2MetricsDB
+    datasource: WSO2MetricsDB
+    max_active: "%{hiera('wso2::datasources::common::max_active')}"
+    max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
+    test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
+    default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
+    validation_query: "%{hiera('wso2::datasources::h2::validation_query')}"
+    validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
+
+#  wso2_metrics_db:
+#    name: WSO2_METRICS_DB
+#    description: The default datasource used for WSO2 Carbon Metrics
+#    driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
+#    url: jdbc:mysql://192.168.100.1:3306/WSO2METRICS_DB?autoReconnect=true
+#    username: "%{hiera('wso2::datasources::mysql::username')}"
+#    password: "%{hiera('wso2::datasources::mysql::password')}"
+#    jndi_config: jdbc/WSO2MetricsDB
+#    datasource: WSO2MetricsDB
+#    max_active: "%{hiera('wso2::datasources::common::max_active')}"
+#    max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
+#    test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
+#    default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
+#    validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
+#    validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
+
+# SSO Configuration
+#wso2::sso_authentication:
+#  enabled: false
+#  login_page: /carbon/admin/login.jsp
+#  service_provider_id: wso2is_analytics
+#  sso_service_url: https://is.dev.wso2.org:9443/samlsso
+#  consumer_service_url: https://is.analytics.dev.wso2.org:9443/acs
+
+# Secure vault configuration
```

Comment:
No Secure Vault values will be written through Puppet anymore.

Reply:
Since the latest released version is 1.0.1 we will be depending on it

Reply:
Ack.